travis.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+
+build_cpp() {
+  ./autogen.sh
+  ./configure
+  make -j2
+  make distcheck -j2
```

Comment:
does "make distcheck" require the previous "make"?  If not, could distcheck be a separate config?